### DISCUSSION TITLE
Repository name "content" is too generic when forked

### DISCUSSION THREAD

**Discussion by SamB** [mdn/mdn-community#592] [Content] [answered]
When I forked it, my fork was at https://github.com/SamB/content, so as soon as I pasted the clone URL after "git clone", I realized that that name was gonna confuse the heck out of me later and figured out how to rename it to https://github.com/SamB/mdn-content.

Now I notice that the instructions literally spell this out for user `octocat`. Poor `octocat` is gonna be so confused later...

Anyway, you might want to rethink this naming; while https://github.com/mdn/mdn-content would look somewhat silly, at least it would lead to clearer fork names.

**Comment by teoli2003** [comment]
Fundamentally, I agree that this would be nice. But: 
- Is this easy to do in github?
- What will be the impact? Will mdn/content#xxyz links in other repo become broken? Will external links to the repo or deeplinks become broken?
- What will happen with repository links locally? Stop working? Being redirected?

I'm afraid that the negative consequences may outnumber the positive ones. But I would like to be proven wrong.

**Reply by Rumyra** [reply]
I agree with you @teoli2003 I think the consequences of re-naming could be unmanageable.

I understand it could be too generic for some - and my apologies for that @SamB - however at the moment we have no plans to update it. If we get more requests like this I will re-assess then 👍

**Comment by peterbe** [comment]
The cloning instructions are to use:

```
git clone git@github.com:octocat/content.git mdn/content
```

that makes it so that you have an automatic prefix/namespace. 
E.g. 
```
cd my-code-projects
git clone git@github.com:octocat/content.git mdn/content
cd mdn/content
```

Perhaps I think it's a bad idea to make your personal fork the "primary". I always let `origin` mean the original repo and `peterbe` is what I name my fork. I.e. **I** do:

```
cd my-code-projects
git clone https://github.com/chalk/supports-color.git
# Now I got to https://github.com/chalk/supports-color and press the "Fork" button,
# then I go back to the command line and type:
git remote add peterbe https://github.com/peterbe/supports-color
```

the advantage of this approach is that the initial `git clone ...` instruction is ready to be copy-n-pasted and you don't need to manually replace "octocat" with "myusername".

**Comment by Rumyra** [comment, marked as answer]
>I understand it could be too generic for some - and my apologies for that @SamB - however at the moment we have no plans to update it. If we get more requests like this I will re-assess then 👍

**Comment by pepelsbey** [comment]
Closing as answered/settled.
